lib/web_of_science/contributions.rb

```diff
@@ -53,7 +53,7 @@ def find_or_create_contribution(author, publication)
     # @param [Author] author
```

Comment:
Commit 3cfd37e07cb9efa8a168f6346c68fdf7a45b93a6 needs to be squashed with the commit that changed the return values.